### DISCUSSION TITLE
What does the green arrow mean on available downloads?

### DISCUSSION THREAD

**Discussion by ExperiencersInternational** [Universal-Team/Universal-Updater#129] [Q&A] [answered]
Sorry if this a stupid question, but what does the green arrow mean next to the app icon on the available downloads section? The app in question is installed on my console, but judging by another discussion article, the installed icon is an SD icon?

When trying to download it, it fails, would assume that you'd have to uninstall the app first.

![16591982228823890522418925592306](https://user-images.githubusercontent.com/56035537/181926158-bc75909c-891a-41c0-a80d-a9c33d2dc5b9.jpg)

**Comment by Epicpkmn11** [comment, marked as answer]
That means that an update has come out since you last installed it. Installing again, updating it, will make that go away.

If you've uninstalled it then tapping the SD card icon will make it stop tracking updates for that app, also making that icon go away.
